int32 X=123 #Ne sert à rien, juste pour pas avoir de paramètre
---
float64 time
